extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/ProxyConfig.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.vertx.http.runtime;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+/**
+ * Holds configuration related with proxy addressing forward.
+ */
+@ConfigGroup
+public class ProxyConfig {
+    /**
+     * Enable override the received request's host through a forwarded host header.
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean enableForwardedForHostHeader;
```

Comment:
This feels a bit long, maybe 'enableForwardedHost' ?

Reply:
Since there exists a header named Forwarded, I would like to see the X in here as well

Reply:
done